src/main/java/edu/byu/ece/rapidSmith/device/creation/XDLRCParseProgressListener.java

```diff
@@ -52,13 +52,14 @@ protected void exitXdlResourceReport(List<String> tokens) {
 	@Override
 	protected void enterTiles(List<String> tokens) {
 		totalTiles = Integer.parseInt(tokens.get(1)) * Integer.parseInt(tokens.get(2));
-		System.out.print(String.format("Parsing tile %6d of %6d tiles.", tilesParsed, totalTiles));
+		System.out.println(String.format("Parsing tile %6d of %6d tiles.", tilesParsed, totalTiles));
 	}
 
 	@Override
 	protected void enterTile(List<String> tokens) {
-		tilesParsed++;
-		System.out.print("\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b\b");
-		System.out.print(String.format("Parsing tile %6d of %6d tiles.", tilesParsed, totalTiles));
+		// print out a heartbeat to the console every 10,000 tiles
```

Comment:
The Installer ran significantly slower in eclipse when you were printing the "\b" characters (the console output was not clearing, it was just printing "\b" over and over). Because of this, eclipse would freeze preventing me from doing any other work. I think a heartbeat printer is a good solution so that the Installer can run on the command line and in an IDE.